pkg/providers/github.go

```diff
@@ -143,6 +151,14 @@ func (g *gitHub) Fetch() (*File, error) {
 func sanitizeName(name, version string) string {
 	name = strings.ToLower(name)
 	replacements := []string{}
+
+	for _, osName := range config.GetOS() {
```

Comment:
Since we're doing a nested loop here, we could move this logic inside here https://github.com/marcosnils/bin/pull/13/files#diff-d73633b9b2f88a927030f7bd9c67fccfR162-R170 so we don't loop again afterwards.

Reply:
The most important part is that we'd check the `_`/`-` + `osName` + `archName` combo first.